worlds/wargroove2/Rules.py

```diff
@@ -0,0 +1,32 @@
+from typing import TYPE_CHECKING
+from worlds.AutoWorld import LogicMixin
+from worlds.wargroove2.Levels import first_level
```

Comment:
Should be a relative import
```suggestion
from .Levels import first_level
```